docs/guides/highlighting_search_terms.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: home
+title: Highlighting search terms
+parent: How-to guides
+nav_order: 7
+---
+
+## Highlighting search terms in results
+
+TIMDEX can return a list of search term matches in the results. The field that returns these matches is called
+'highlight':
+
+```graphql
+{
+  search(searchterm: "data") {
+    records {
+      highlight {
+        matchedField
+        matchedPhrases
+      }
+    }
+  }
+}
+```
+
```

Comment:
What are your thoughts on having a "[Run this query in the Playground](/playground?query=%7B%0A%20%20%20search(searchterm%3A%20%22data%22)%20%7B%0A%20%20%20%20%20records%20%7B%0A%20%20%20%20%20%20%20highlight%20%7B%0A%20%20%20%20%20%20%20%20%20matchedField%0A%20%20%20%20%20%20%20%20%20matchedPhrases%0A%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%7D%0A%20%20%20%7D%0A%20%7D)" link for each example?

I generally think it will be good to do that but would be okay with coming back to that later as well.

Reply:
On second thought, there's nothing stopping us from using `timdex.mit.edu`. The queries just won't work until we launch, which I think is fine.

Reply:
Good point. On launch we'll link prominently to these docs. Before that they are hidden behind having to know the link which I suspect most people aren't interested enough to dig up.